analyses/molecular-subtyping-pathology/03-incorporate-pathology-feedback.Rmd

```diff
@@ -526,6 +526,68 @@ compiled_df <- compiled_df %>%
   )
 ```
 
+
+### Chordoma samples pathology review 
+As discussed in [ticket](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/608) we will assign subtypes to chordoma samples through pathology review only since molecular subtypes were not sufficient.
+	
+ sample_id | molecular_subtype
+-- | --
+7316-1101 |	CHDM, poorly differentiated
+7316-2935 |	CHDM, poorly differentiated
+7316-3295 |	CHDM, poorly differentiated
+7316-4062 |	CHDM, poorly differentiated
+7316-921 |	CHDM, conventional
+7316-3632	| CHDM, conventional
+
+```{r}
+compiled_df %>%
+  filter(sample_id %in% c("7316-1101",
```

Comment:
Given the number of times you repeat these sample IDs, it might make sense to make a vector out of them and make a data frame with this additional info you are coding it by.

Reply:
ok will do, I'll add a vector of sample_ids per molecular_subtype values here.